### ISSUE TITLE
README.md leads to deadlink

### ISSUE THREAD

**Issue by codejunkie84** [opis/utils#1]
In the README.md is a link contained to the documentation of `opis/utils` but this leads to a github 404 page.

http://www.opis.io/utils

The same issue exists on the http://www.opis.io/ page, when you click on "documentation" for `opis/utils`.

**Comment by msarca** [comment]
Yeah, sorry about that. This library is not under active development anymore. We're keeping it just because some old customer is still using it. We are about to rebuild and improve our website and a proper cleanup will be made once we are done with rebuilding. Thanks for letting us know.
